scripts/install.sh

```diff
@@ -35,7 +35,29 @@ else
     exit 1
 fi
 
-# TODO(nick): Add verification that Tilt installed successfully.
-
 set +x
-echo "Tilt installed! Run \`tilt up\` to start."
+
+
+VERSION="$(tilt version 2>&1 || true)"
+RUBY_TILT_PATTERN="template engine not found"
+TILT_DEV_PATTERN='^v[0-9]+\.[0-9]+\.[0-9]+(-dev)?, built [0-9]+-[0-9]+-[0-9]+$'
```

Comment:
oh boy, this regexp seems very likely to break. can you add a ci job that verifies this works? maybe something that just runs the install script and checks the exit code?

Reply:
good idea!